spacialgaze-plugins/misc.js

```diff
@@ -433,4 +433,26 @@ exports.commands = {
 		Db.disabledScrolls.remove(target);
 	},
 	enableintroscrollhelp: ["/enableintroscroll [room] - Enables scroll bar preset in the room's roomintro."],
+
+	'!ai': true,
+	askai: 'ai',
+	'8ball': 'ai',
+	ai: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		const http = require('http');
```

Comment:
Plus this command will probably become spammable.... so you know we want it to be usable multiple times with minimum inconvenience